advanced-coroutines-codelab/finished_code/src/main/java/com/example/android/advancedcoroutines/PlantListViewModel.kt

```diff
@@ -127,10 +123,7 @@ class PlantListViewModel internal constructor(
      */
     fun setGrowZoneNumber(num: Int) {
         growZone.value = GrowZone(num)
-        growZoneChannel.offer(GrowZone(num))
-
-        // initial code version, remove during flow rewrite
-        launchDataLoad { plantRepository.tryUpdateRecentPlantsForGrowZoneCache(GrowZone(num)) }
+        growZoneFlow.value = GrowZone(num)
```

Comment:
Nit: it's a bit confusing having `growZone` as LiveData and `growZoneFlow` as StateFlow, and then calling `.value` to both of them one after the other. Not sure what's the difference at first glance to be honest. But if it makes sense in the narrative of the codelab, leave it as is :)

Reply:
Yeah, this is an artifact of having both LiveData and Flow in the same codelab.
I'll fix it in a next set of commits and codelab text changes